README.md

```diff
@@ -41,52 +41,148 @@ You can follow the template advice or not. But it could be better if you do.
                 └─zh_CN
 ```
 
-## :sparkles: Features
+## :open_file_folder: README.md
+
+The Github README markdown file for your plugin project.
+
+## :open_file_folder: LICENSE
+
+Update this file to outline the license for your plugin.
+
+## :open_file_folder: plugin.json
+
+The file that Flow uses to incorporate your plugin to the plugin manifest list. See [https://github.com/Flow-Launcher/Flow.Launcher.Plugin.PythonTemplate/tree/master#commands.py].
+
+## :open_file_folder: .gitignore
+
+Files for git to ignore when merging changes.
+
+## :open_file_folder: .env
+
+User config file. Currently sets the language for the plugin. The user will need to manually edit this once the plugin is installed to change the language (if you have localized the plugin. See [https://github.com/Flow-Launcher/Flow.Launcher.Plugin.PythonTemplate/tree/master#commands.py]).
```

Comment:
The link has the branch in it. If I add the correct link for this 'readme-update' branch does it get auto updated when the PR is merged with 'master' branch?

Reply:
Actually, it can't.

if there has emoji, the link should be 

![image](https://user-images.githubusercontent.com/25895405/121305288-fff7ae00-c92f-11eb-9728-3b0141955785.png)

if there has `.`, the link should be

![image](https://user-images.githubusercontent.com/25895405/121305435-24ec2100-c930-11eb-9a7f-5bbc2616f55c.png)

Reply:
I figured out relative references and have updated the links